content/articles/top-container-orchestration-tools/index.md

```diff
@@ -0,0 +1,137 @@
+- [Top container orchestration tools](#top-container-orchestration-tools)
+
+### Introduction to container orchestration
+To understand the problem solved by [container orchestration](https://docs.docker.com/get-started/orchestration/). Think of a case where you have two containers:
+- A front-end application container
+- A back-end application container
+
+These two containers need to work together as one. The solution to the above problem is container orchestration.
+
+Container orchestration is the automation of all processes of organizing and managing containers. Container orchestration automates deploying, scheduling, management, and networking of containers. Organizations that deploy, manage and host containers should use container orchestration. It helps you to deploy the application in different environments seamlessly.
+
+Microservices in containers make it straightforward to orchestrate services such as scalability, security, storage, flexibility, and networking. Containers help us combine applications to development, deployment, management, and running. At the same time, we need a set of tools to manage these containers. This is where container orchestration tools come in handy.
+
+### Why container orchestration is important
+Container Orchestration manages the life cycle of containers in dynamic environments. Developers use container orchestration to automate and manage tasks in an application. While containers help improve the developer's productivity.  Container orchestration benefits the enterprises by optimizing their DevOps and operational activities.
+
+Orchestration manages and automates tasks that include:
+- Configuration and scheduling
+- Provision and deployment
+- Resource allocation
+- Containers availability
+- Monitoring the health of container
+- Securing interactions between containers
+- Load balancing, service routing, and traffic routing between containers
+- Adding or removing containers to balance the workload 
+- Moving containers from one host to another if resources are limited or when a host shut down
+
+### Top container orchestration tools
+Container orchestration tools manage many containers and microservices for your organization.
+There are multiple container orchestration tools used for container management. This article will discuss popular tools, Kubernetes, Docker Swarm, Apache Mesos, and others.
+
+#### Kubernetes
+[Kubernetes](https://kubernetes.io/) is an open-source platform that was designed and developed by Google. Currently, Kubernetes is maintained by [Cloud Native Computing Foundation](https://www.cncf.io/). It is the leading container orchestration system. This is because container solutions widely adopt it as it is user-friendly.
+Kubernetes automates deployment, scaling, and management of containerized applications.
+
+![Kubernetes Architecture](/engineering-education/top-container-orchestration-tools/kubernetes-architecture.jpg)
```

Comment:
You have specified the name of the image file correctly as `kubernetes-architecture.jpg`. But, the image file is named as `kubernetes architecture.jpg`. (Space instead of hyphen - Don't use space)

Could you please rename the image file as `kubernetes-architecture.jpg`?